src/components/Nav.jsx

```diff
@@ -37,8 +36,8 @@ function Nav() {
             <NavWrap className="form-wrap">
         <NavLink  id ="home" href="https://expactmarketing.netlify.app/#main">Home</NavLink>
         <Link  id ="login" exact to ='/login'>Login</Link>
-        <Button href ='/signup'>signup</Button>
-</NavWrap>
+        <Link to='/signup'><Button href ='/signup'>signup</Button></Link>
```

Comment:
When routing you don't want to nest a `button` inside of a `link` or `a` element